region/crateria/west/Gauntlet Entrance.json

```diff
@@ -371,6 +371,26 @@
       ],
       "devNote": "TODO: There is a blue SpringBall strat to investigate."
     },
+    {
+      "link": [1, 2],
+      "name": "Gauntlet Entrance Blue SpaceJump (Left to Right, Come in Blue Spinning)",
+      "notable": true,
+      "entranceCondition": {
+        "comeInBlueSpinning": {
```

Comment:
What I was really looking for is cases where you have to fall to reach the door and then can't spacejump in the next room.
LN elevator room has something like that, but you can probably always jump into the top of the door and be setup to continue spacejumping from there.

Reply:
If there were a case where you could only fall through the door like that, then I think we wouldn't put "leaveSpinning" on it (at least not without restricting it by some new property that we could define on it), because that exit condition is supposed to imply a greater degree of flexibility in how you can jump through the door.